README.md

```diff
@@ -94,6 +94,28 @@ client.basicGet("my.queue", true, getResult -> {
 });
 ```
 
+## Consume messages without auto-ack
```

Comment:
At some point we would need to switch to the "way" vert.x stack is handling documentation.